index.html

```diff
@@ -19,7 +19,7 @@
       integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
       crossorigin="anonymous"
     />
-    <title>Protfolio</title>
+    <title>Mohamed's Portfolio homepage</title>
```

Comment:
- [ ] If you click on line 22 or 23, or any other line, you can see the URL changing. For example, by clicking line 22 we can see the URL https://github.com/eng-mohamed-eldeeb/protofolio/pull/5/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L22. You can copy this link for your report, for your the first point:

- Page titles - one issue found [link](https://github.com/eng-mohamed-eldeeb/protofolio/pull/5/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L22)

Please, consider this technic for all found issues.

Reply:
thank you

Reply:
@eng-mohamed-eldeeb You are welcome.

Reply:
Please, we have made all the changes you requested.

Reply:
@AckonSamuel Nice job done 👍